fe/fe-core/src/main/java/org/apache/doris/analysis/Expr.java

```diff
@@ -2559,7 +2559,7 @@ protected Type[] getActualArgTypes(Type[] originType) {
     }
 
     private MapType getActualMapType(MapType originMapType) {
-        return new MapType(getActualType(originMapType.getKeyType()), getActualType(originMapType.getValueType()));
+        return new MapType(originMapType.getKeyType(), originMapType.getValueType());
```

Comment:
do array and struct have same problem?

Reply:
maybe not, array some functions not has template type

Reply:
struct has not, just return origin type